custom_components/adjustable_bed/config_flow.py

```
@@ -163,6 +164,25 @@ def __init__(self) -> None:
+        """Prepare the focused bed-type chooser for an ambiguous detection."""
+        bed_type = detection_result.bed_type
+        if (
+            bed_type is None
+            or detection_result.confidence >= 0.7
+            or not detection_result.ambiguous_types
+        ):
+            return False
+
+        seen: set[str] = set()
+        disambiguation_types: list[str] = []
+        for candidate in [bed_type] + list(detection_result.ambiguous_types):
```

Ruff (0.15.14)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)